best-practices/README.md

```diff
@@ -329,3 +329,18 @@ Angular
 [annotations]: http://robots.thoughtbot.com/avoid-angularjs-dependency-annotation-with-rails
 [ngannotate]: https://github.com/kikonen/ngannotate-rails
 [angular-translate]: https://github.com/angular-translate/angular-translate/wiki/Getting-Started#using-translate-directive
+
+Ruby JSON APIs
+--------------
+
+* Review the recommended practices outlined in Heroku's [HTTP API Design Guide]
+  before designing a new API.
+* Use a fast JSON parser, e.g. [`oj`][oj]
```

Comment:
@harlow good to know, thank you! I went ahead and merged this as-is but will see about expanding the recommendation to use `oj` to include some caveats about its usage in a separate PR.